actions/verify-helm-schema/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
       uses: giantswarm/install-binary-action@v1.1.0
       with:
         binary: "schemalint"
-        version: "2.2.1"
+        version: "2.3.0-alpha.0"
```

Comment:
This is for testing purposes during cluster apps refactoring. See https://github.com/giantswarm/cluster-aws/pull/413. It will be removed before merging the PR.

Reply:
Reverted.